docs/DeveloperGuide.md

```diff
@@ -103,119 +108,155 @@ Format: `search -title TITLE`
 Example:  
 - `search -title Python Programming`  
 
-For successful searches, the program will output the relevant book along with the book details: ISBN, Title, Author, Topic. For unsuccessful searches, the program will output a string informing the user that there is no match with the input title from the inventory.  
-Note: Partial string matching will not be considered in this application. Exact match of title and topic will be done, and if match is found, the book object will be returned.
+Successful searches will result in the program outputting the relevant book details, including ISBN, 
+title, author, and topic. If the search is unsuccessful, the program will output a message indicating 
+that there is no match in the inventory for the input title. 
+
+Note: Partial string matching will not be considered in this application. 
+The search will only return exact matches for the book's title and topic. If a match is found, 
+the book object will be returned.
```

Comment:
This is no longer valid as we have fixed this problem. I will be creating another PR to fix this issue